### ISSUE TITLE
_.contains is not a function

### ISSUE THREAD

**Issue by kmaraz** [MathieuLoutre/grunt-aws-s3#128]
With the new version 2.0.1 I have got an error during file upload which says:

Warning: _.contains is not a function

```
grunt.initConfig({
    aws_s3: {
      options: {
        region: 'eu-west-1',
        uploadConcurrency: 5,
        downloadConcurrency: 5,
        differential: false
      },
      upload_immutables: {
        options: {
          bucket: process.env.APP_BUCKET,
          differential: false
        },
        files: getImmutableFiles("/admin")
      }
    }
  });

  grunt.loadNpmTasks('grunt-aws-s3');
```

**Comment by tirish** [comment]
We also had this issue. Had to downgrade to 2.0.0 which resolved it for us.

**Comment by bkelsey** [comment]
Also having this issue. Can confirm forcing 2.0.0 fixes the issue at least for now.

**Comment by peterpod-c1** [comment]
I still get the issue with version 2.0.0. How does downgrading fix the problem if lodash version is still v4

**Comment by justinwinslow** [comment]
https://github.com/MathieuLoutre/grunt-aws-s3/blob/master/tasks/aws_s3.js#L85

This line just needs to be changed to `_.includes`.

**Comment by lifenstein** [comment]
> I still get the issue with version 2.0.0. How does downgrading fix the problem if lodash version is still v4

It's not... The lodash version bump from 2.x to 4.x was the only change in the 2.0.1 patch release.

**Comment by justinwinslow** [comment]
You can use this until the owner updates: https://github.com/justinwinslow/grunt-aws-s3

**Comment by evangboucher** [comment]
Hi @MathieuLoutre 
Any update on @justinwinslow's PR? This is a blocker for us using the newest version of the library that updates to Lodash V4. Unfortunately Lodash V2 has security vulnerabilities which we were hoping to remove from our dependency tree. Thanks in advance! 🙇

**Comment by MathieuLoutre** [comment]
I've been alerted via email of this issue. I've published 2.0.2 which should fix this. I don't use Grunt anymore (or this package) so if someone would like to maintain it, please let me know.

**Comment by lifenstein** [comment]
@MathieuLoutre Please make a new release so that v2.0.2 gets picked up by npm.

**Comment by MathieuLoutre** [comment]
@lifenstein I've already published 2.0.2 on npm: https://www.npmjs.com/package/grunt-aws-s3
